corehq/apps/app_manager/templates/app_manager/partials/build_errors.html

```diff
@@ -39,6 +39,12 @@ <h4 class="alert-heading">
         {% blocktrans %}
           Please check that end of form navigation is correct. It appears to be pointing
           to a parent menu form, but the menu doesn't have a parent.
+         {% endblocktrans %}
+        {% include "app_manager/partials/form_error_message.html" %}
+          {% case "form link to display only forms" %}
```

Comment:
@biyeun fyi 96dce1cc722b0966156c663fe1545bedb1667297 seems to have borked this file's indentation - maybe pycharm doesn't understand case statements in django templates?